gobblin-service/src/main/java/org/apache/gobblin/service/modules/utils/InjectionNames.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gobblin.service.modules.utils;
+
+public final class InjectionNames {
```

Comment:
What's the reason for this additional class?

Reply:
These names are used for dependency injection, when we need to inject different instances of the same type, or inject constants. Since they are used in multiple classes, they don't fully belong to any of them, and that's why they landed in a separate class. FLOW_CATALOG_LOCAL_COMMIT for example is used in GobblinServiceFlowConfigResourceHandler and GobblinServiceFlowConfigV2ResourceHandler.